### PR TITLE
Updating update-swagger-spec to build the apiserver before running

### PR DESCRIPTION
**Pull request by nikhiljindal**
Fixes https://github.com/GoogleCloudPlatform/kubernetes/issues/11432

### CODE REVIEW

**Comment by nikhiljindal** [comment]
cc @thockin @zmerlynn

**Comment by nikhiljindal** [comment]
cc @davidopp

File: hack/update-swagger-spec.sh [hunk 1]
```diff
@@ -36,6 +36,9 @@ function cleanup()
 
 trap cleanup EXIT SIGINT
 
+kube::log::status "Building apiserver"
+"${KUBE_ROOT}/hack/build-go.sh" cmd/kube-apiserver
```
Comment (thockin) [inline hack/update-swagger-spec.sh]: LGTM


we should have a --dump-swagger=VER flag, rather than running apiserver

**Comment by thockin** [comment]
abhishek is away, travis passed with this.

**Comment by thockin** [comment]
merging

**Pull request merged**